source/UberCore/Authentication/LoginManager.swift

```diff
@@ -40,16 +41,18 @@ import SafariServices
     Create instance of login manager to authenticate user and retreive access token.
     
     - parameter accessTokenIdentifier: The access token identifier to use for saving the Access Token, defaults to Configuration.shared.defaultAccessTokenIdentifier
-    - parameter keychainAccessGroup:   The keychain access group to use for saving the Access Token, defaults to Configuration.shared.defaultKeychainAccessGroup
-    - parameter loginType:         The login type to use for logging in, defaults to Implicit
-    
+    - parameter keychainAccessGroup: The keychain access group to use for saving the Access Token, defaults to Configuration.shared.defaultKeychainAccessGroup
+    - parameter loginType: The login type to use for logging in, defaults to Implicit
+    - parameter productFlowPriority: An ordered list of the Uber apps to use for authentication, if available. Defaults to trying to use only the main Uber rides flow.
+
     - returns: An initialized LoginManager
     */
-    @objc public init(accessTokenIdentifier: String, keychainAccessGroup: String?, loginType: LoginType) {
+    @objc public init(accessTokenIdentifier: String, keychainAccessGroup: String?, loginType: LoginType, productFlowPriority: [UberAuthenticationProductFlow] = [UberAuthenticationProductFlow(.rides)]) {
```

Comment:
The reason why we have all these convenience inits is that objc won't see the default values provided by `productFlowPriority: [UberAuthenticationProductFlow] = [UberAuthenticationProductFlow(.rides)]`. So I would suggest not putting default values here, and instead create more convenience inits.